packages/terra-arrange/docs/Arrange.md

```diff
@@ -0,0 +1,9 @@
+| Prop Name | Type | Is Required | Default Value | Description |
```

Comment:
I'd like to use it both as doc on GitHub as well as import it into the examples so it can on be used on the doc site.

The concern Matt noted about keeping the props table in sync with the src code when imported into the examples would be a good topic for Weds. meeting.